src/test/java/com/github/iveina2021/automationexercises/LoginPage.java

```diff
@@ -0,0 +1,36 @@
+package com.github.iveina2021.automationexercises;
+
+import com.codeborne.selenide.SelenideElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.UUID;
+
+import static com.codeborne.selenide.Selenide.page;
+
+// page_url = https://automationexercise.com/login/
+
+public class LoginPage {
+
+    @FindBy(css = "div[class='signup-form'] h2")
+    public SelenideElement newUserSignupLabel;
+
+    @FindBy(css = "input[data-qa='signup-name']")
+    public SelenideElement signupNameInput;
+
+    @FindBy(css = "input[data-qa='signup-email']")
+    public SelenideElement signupEmailInput;
+
+    @FindBy(css = "button[data-qa='signup-button']")
+    public SelenideElement signupButton;
+
+    public void checkNewUserSignUpLabelIsVisible() {
+        newUserSignupLabel.isDisplayed();
+    }
+
+    public SignUpPage fillNewUserSignUpForm() {
```

Comment:
'Leonard' is the hardcoded value here. Users of this class (PageObject) may want to specify different values for different scenario.

So it's better to introduce parameter 'name' to 'fillNewUserSignUpForm' method. (and maybe other parameters too)

Reply:
Edited it for all parameters